pycard/card.py

```diff
@@ -15,18 +15,25 @@ class Card(object):
     BRAND_MASTERCARD = 'mastercard'
     BRAND_AMEX = 'amex'
     BRAND_DISCOVER = 'discover'
+    BRAND_MAESTRO = "maestro"
+    BRAND_JCB = "jcb"
     BRAND_UNKNOWN = u'unknown'
     BRANDS = {
         BRAND_VISA: re.compile(r'^4\d{12}(\d{3})?$'),
         BRAND_MASTERCARD: re.compile(r'^(5[1-5]\d{4}|677189)\d{10}$'),
-        BRAND_AMEX: re.compile(r'^3[47]\d{13}$'),
+        BRAND_AMEX: re.compile(r'^3[4,7]\d{13}$'),
```

Comment:
@onegreyonewhite sorry for the delay on this - thanks for the patch.

Would you mind explaining this (the comma addition)?

The pattern from http://www.regular-expressions.info/creditcard.html is:

    3[47][0-9]{13}